sbin/celeborn-daemon.sh

```diff
@@ -166,6 +166,11 @@ run_command() {
 }
 
 start_celeborn() {
+    if [ ! -d "$CELEBORN_CONF_DIR" ]
+    then
+      echo "ERROR : CELEBORN_CONF_DIR: $CELEBORN_CONF_DIR is not a directory"
+      exit 1
+    fi
```

Comment:
The celeborn-daemon.sh would load the celenorn-env.sh after check the `--config` option.
https://github.com/apache/celeborn/blob/fde6365f68ce8b589bcc639e96476ed618249257/sbin/celeborn-daemon.sh#L79

So we just check it before run the start command.